### PR TITLE
🎬 Notifications take 2

### PR DESCRIPTION
**Pull request by rowanseymour**
The whole `Log` v `Notification` separation seemed like a good idea but was going to make it tricky to efficiently find the notifications that need to be marked as seen when users go places, and know when to create new notifications for specific users

### CODE REVIEW

File: temba/notifications/models.py [hunk 1]
```diff
@@ -144,96 +103,119 @@ class Log(models.Model):
...
+class Notification(models.Model):
+    """
+    A user specific notification
+    """
 
-        log_type = TYPES_BY_SLUG[log_type]
-        log = cls.objects.create(org=org, created_by=user, log_type=log_type.slug, **kwargs)
+    id = models.BigAutoField(primary_key=True)
+    org = models.ForeignKey(Org, on_delete=models.PROTECT, related_name="notifications")
+    notification_type = models.CharField(max_length=16, null=True)
```
Comment (norkans7) [inline temba/notifications/models.py:114]: is 16 enough here?

Comment (rowanseymour) [reply temba/notifications/models.py:114]: I think we can keep them under 16..  save space

Comment (norkans7) [reply temba/notifications/models.py:114]: "broadcast:finished" could be a notification, may be 32 or 36 will give room for such

Comment (rowanseymour) [reply temba/notifications/models.py:114]: when I previously had notification types for broadcasts, I called them `bcast:xxx`

Comment (rowanseymour) [reply temba/notifications/models.py:114]: it's easy to make it bigger later if we really want to be more verbose

**Review by norkans7 — APPROVED** [review]

**Pull request merged**